build/build.yml

```diff
@@ -101,6 +94,32 @@ steps:
     targetFolder: '$(Build.ArtifactStagingDirectory)\tarball\'
     flattenFolders: true
 
+- task: ArchiveFiles@2
+  displayName: Create netcore tarball
+  inputs:
+    rootFolderOrFile: '$(Build.ArtifactStagingDirectory)\tarball\'
+    includeRootFolder: false
+    archiveType: 'tar'
+    tarCompression: 'gz'
+    archiveFile: '$(Build.ArtifactStagingDirectory)\$(BuildConfiguration)\Microsoft.NetCore.NuGet.CredentialProvider.tar.gz'
+    replaceExistingArchive: true
+
+- task: ArchiveFiles@2
+  displayName: Create netcore zip
+  inputs:
+    rootFolderOrFile: '$(Build.ArtifactStagingDirectory)\tarball\'
+    includeRootFolder: false
+    archiveType: 'zip'
+    archiveFile: '$(Build.ArtifactStagingDirectory)\$(BuildConfiguration)\Microsoft.NetCore.NuGet.CredentialProvider.zip'
+    replaceExistingArchive: true
+
+- task: CopyFiles@2
+  displayName: Copy netfx files into tarball
+  inputs:
+    sourceFolder: '$(Build.SourcesDirectory)\CredentialProvider.Microsoft\bin\$(BuildConfiguration)\net461\'
```

Comment:
net461 hardcoded? I assume it'll at least break the build if we upgraded netfx ver?

Reply:
good catch! will fix this as weel in follow up PR.